Lib/enum.py

```diff
@@ -538,7 +540,8 @@ def __new__(metacls, cls, bases, classdict, *, boundary=None, _simple=False, **k
         classdict['_member_names_'] = []
         classdict['_member_map_'] = {}
         classdict['_value2member_map_'] = {}
-        classdict['_unhashable_values_'] = []
+        classdict['_hashable_values_'] = []          # for comparing with non-hashable types
+        classdict['_unhashable_values_'] = []       # e.g. frozenset() with set()
```

Comment:
It's best not to use Latin, which isn't convenient for unfamiliar users.

Reply:
Sorry, but what do you mean by saying "It's best not to use Latin"? What is an the alternative approach?

Reply:
In fact, `e.g.` It is equivalent to `for example` in english.
This will be more intuitive.

Reply:
At some point I'll need to go through and standardize `enum.py`, but not in this PR.